ModelBundle/Repository/SiteRepository.php

```diff
@@ -118,13 +118,17 @@ public function findByAliasDomain($domain)
             )
         ));
 
+        $ids = array();
         if (is_array($commandResult) && array_key_exists('ok', $commandResult) && $commandResult['ok'] == 1) {
             foreach ($commandResult['results'] as $siteId) {
-                return $this->findOneBySiteId($siteId['_id']);
+                $ids[] = $siteId['_id'];
```

Comment:
We need to find a site which contains a site-alias matching the domain.We don't need all the sites but just the first one. That's was made by the return working as a break or as a first.